.gitignore

```diff
@@ -2,58 +2,77 @@
 logs
 *.log
 npm-debug.log*
-.[Dd][Ss]_[Ss]tore
-
-# Ignore test results
-**/test/results/*
+yarn-debug.log*
+yarn-error.log*
 
 # Runtime data
 pids
 *.pid
 *.seed
+*.pid.lock
 
 # Directory for instrumented libs generated by jscoverage/JSCover
 lib-cov
 
 # Coverage directory used by tools like istanbul
 coverage
 
-#Docs folder created for documentation
-docs
-
 # nyc test coverage
 .nyc_output
 
 # Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
 .grunt
 
+# Bower dependency directory (https://bower.io/)
```

Comment:
wow, I didn't notice we are using bower :-D

Reply:
Must of been a catchall sort of ignore in the auth-service.
done.